static/index.scss

```diff
@@ -58,9 +64,7 @@
       text-align: center;
       font-size: 1.5em;
       width: 100%;
-      padding-top: 0.25em;
-      padding-bottom: 0.25em;
-      margin-top: 0em;
+      padding-top: 0.25em 0;
```

Comment:
`padding-top` should only have 1 value. Your options are 
```css
padding-top: .25em;
padding-bottom: .25em;
```
or
```css
padding: .25em 0
```
Where `.25em` stands for top and bottom and `0` stands for left and right.